.buildkite/scripts/common/env.sh

```diff
@@ -9,7 +9,7 @@ export KIBANA_DIR
 export XPACK_DIR="$KIBANA_DIR/x-pack"
 
 export CACHE_DIR="$HOME/.kibana"
-export ES_CACHE_DIR="$HOME/.es-snapshot-cache"
+export ES_SNAPSHOT_MANIFEST="https://storage.googleapis.com/kibana-ci-es-snapshots-daily/9.0.0/archives/20241111-031745_fae80f85/manifest.json"
```

Comment:
Temporary change. Waiting for the CI as suggested by @jbudz.

Reply:
Also ran it locally it could already see the logs from Kibana API response:

```
{
           │        statusCode: 404,
           │        error: 'Not Found',
           │        message: 'No indices match "basic_index"',
           │        attributes: {
           │          statusCode: 404,
           │          error: 'Not Found',
           │          message: 'No indices match "basic_index"',
           │          code: 'no_matching_indices'
           │        }
           │      }
```

Then I executed the following in DevTools:
```
PUT test
PUT test/_mapping
{
  "properties": {
    "actual": {
      "type": "keyword"
    }
  }
}
POST test/_field_caps?fields=*
POST test/_close
POST test/_field_caps?fields=*
```

And got the following response from ES:
<img width="1457" alt="Screenshot 2024-11-11 at 18 14 32" src="https://github.com/user-attachments/assets/9ec4ee08-2679-4ebf-8fdd-75e3e40d7c4e">

It used to be `cluster_block_exception` before:
<img width="1460" alt="Screenshot 2024-11-11 at 18 14 12" src="https://github.com/user-attachments/assets/1ea2e23a-269e-49f1-8b1e-8d5417c9c6b9">